src/khoj/routers/helpers.py

```diff
@@ -542,8 +561,7 @@ def send_message_to_model_wrapper_sync(
         )
 
     elif conversation_config.model_type == "openai":
-        openai_chat_config = ConversationAdapters.get_openai_conversation_config()
-        api_key = openai_chat_config.api_key
+        api_key = conversation_config.openai_config.api_key
```

Comment:
Should we rename `openai_config` to something more generic, e.g `online_model`? 

```suggestion
        api_key = conversation_config.online_model.api_key
```